R/tax_certainty.R

```diff
@@ -0,0 +1,33 @@
+#' Check if a taxon is certain or uncertain.
+#'
+#' Check whether a given taxon in a vector is certain or uncertain.
+#' Binomial names (genus and species) are considered "certain",
+#' whereas names containing only the genus,
+#' or genus and not determined species is considered "uncertain".
+#'
+#' @param taxon_names Character vector containing the species names to be
+#' checked.
+#' It allows taxon names separated by underscore or space.
+#' @return Numeric vector classifying each species as certain (1) or uncertain
+#' (0).
+#' @examples
+#' sirenia_sp <- c("Metaxytherium indet.", "Felsinotherium_serresii",
+#'                  "Felsinotherium gervaisi", "Potamosiren indet.",
+#'                  "Potamosiren", "Metaxytherium sp.")
+#'
+#' certainty <- tax_certainty(sirenia_sp)
+#'
+#' @export
+tax_certainty <- function(taxon_names) {
```

Comment:
I am sure this wouldn't happen frequently, but I noticed that if for some reason, a user supplies a vector of just length 1, the function fails. e.g. 
```
taxon_names <- c("Metaxytherium indet.")
tax_certainty(taxon_names)
```